tests/WeatherParserUtilsTests.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const WeatherParserUtils = require('../lib/Position/WeatherParserUtils.js');
+
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('WeatherParserUtils', () => {
+    it('Text regex & extract comment from weather', () => {
```

Comment:
Could you please add assertions on other weather fields as well. Ideally we should cover all fields and case with negative temperature.